crawl-ref/source/spl-summoning.cc

```diff
@@ -948,10 +949,14 @@ spret_type cast_summon_lightning_spire(int pow, const coord_def& where, god_type
         fail_check();
 
         // invisible monster
-        mpr("Something you can't see is blocking your construction!");
+        canned_msg(MSG_GHOSTLY_OUTLINE);
```

Comment:
Switched this to use the canned_msg that f. prism uses. Seemed like a reasonable thing to make them behave the same way.